.github/workflows/on_pull_request.yml

```diff
@@ -229,27 +229,24 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-yarn
 
-      - name: Node modules cache
-        uses: actions/cache@v2
-        id: node_modules_cache_id
-        with:
-          path: |
-            node_modules
-            */*/node_modules
-          key: ${{ runner.os }}-${{ hashFiles('**/yarn.lock') }}
-
       - name: Install Dependencies
-        if: steps.yarn_cache_id.outputs.cache-hit != 'true' || steps.node_modules_cache_id.outputs.cache-hit != 'true'
```

Comment:
suggestion: we might want to leave a little inline comment here reminding us why we don't want the cached node modules for this job. Otherwise we might forget and think it was a mistake.

Reply:
great point. Comment left